### PR TITLE
blocky 0.21 (new formula)

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

https://github.com/0xERR0R/blocky

### CODE REVIEW

File: Formula/blocky.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          - 1.0.0.1
+      bootstrapDns:
+        - tcp+udp:1.1.1.1
+        - https://1.1.1.1/dns-query
+      log:
+        level: warn
+        format: text
+        timestamp: true
+        privacy: true
+    EOF
```
Comment (SMillerDev) [inline Formula/blocky.rb:39]: This should be sourced from upstream, not written in the formula

File: Formula/blocky.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X github.com/0xERR0R/blocky/util.Version=#{version}
+      -X github.com/0xERR0R/blocky/util.BuildTime=#{time.iso8601}
+    ]
+    system "go", "build", *std_go_args(ldflags: ldflags)
+    sbin.install_symlink bin/"blocky"
```
Comment (SMillerDev) [inline Formula/blocky.rb:18]: Better do 
```suggestion
    system "go", "build", *std_go_args(ldflags: ldflags, output: sbin/"blocky")
```

Comment (log-to-code) [reply Formula/blocky.rb]: Thanks!

Comment (log-to-code) [reply Formula/blocky.rb]: Do we have to update the binary path for the client test?

Comment (log-to-code) [reply Formula/blocky.rb]: The binary path for the client test has been changed to sbin.

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**